docs/software/backpack-telemetry.md

```diff
@@ -0,0 +1,72 @@
+---
+template: main.html
+description: ExpressLRS Backpack can forward telemetry data to third party systems via ESPNOW.
+---
+
+![Software Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/software.png)
+
+## Introduction
+Version 3.4+ of ELRS introduces a new option in the Backpack section of the ELRS LUA script called "Telemetry." This feature allows a Backpack equipped ELRS Transmitter to forward CRSF telemetry frames to any peer that is capable of receiving ESPNOW messages. These frames can be consumed by any Backpack receivers that are bound with your bind phrase.
+
+<figure markdown>
+![Sentinel Integration](../assets/images/backpack-telemetry.png)
+<figcaption>Sentinel Integration</figcaption>
+</figure>
+
+This capability is particularly useful for devices like the Sentinel Antenna Tracker, which can subscribe to ESPNOW telemetry packets and read GPS data from the telemetry feed. See https://www.virtualpilot.co.uk/index.php?route=product/product&product_id=59
+
+## Prerequisites
+- ELRS transmitter and receiver with the latest firmware.
+- Backpack-capable receiver (e.g., in the Sentinel Antenna Tracker).
+- Configured ELRS LUA script on your transmitter.
+
+## Setting Up the Backpack Telemetry Feature
+
+### Step 1: Update Firmware
+Ensure both your transmitter and receiver are running the latest firmware. Follow the [firmware update guide](https://www.expresslrs.org/software/quick-start/) for detailed instructions. This feature is available in v3.4.0 onwards.
```

Comment:
url needs correction
please use ../quick-start/getting-started.md

Reply:
The list isn't being formatted correctly.

![wezStep-by-Step](https://github.com/ExpressLRS/Docs/assets/83231715/12c425b7-4e2e-4105-a174-c0a0b71ae648)

Add another indent on the sub list and that paragraph under no.4

![wezStep-by-Step_fix](https://github.com/ExpressLRS/Docs/assets/83231715/513dff1a-d969-4e43-b7b9-3f001546ca5a)